librz/util/json_parser.c

```diff
@@ -567,3 +568,56 @@ RZ_API RZ_OWN char *rz_json_as_string(const RzJson *json, bool with_key) {
 	json_pj_recurse(json, pj, with_key);
 	return pj_drain(pj);
 }
+
+RZ_API bool rz_json_eq(const RzJson *a, const RzJson *b) {
```

Comment:
Please add Doxygen and a small test in `test/unit/test_json.c` testing both equality and inequality.

Reply:
Please add `RZ_NONNULL` and `rz_return_val_if_fail()` as well